### PR TITLE
Travis-CI Integration

### PR DESCRIPTION
**Pull request by aaryanrr**
- Info about Issue or bug

Closes: #46 

The YAML Configuration file for Travis-CI has been added. The build will be triggered each time a PR is merged with the master branch. 

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  - "3.8"
+  - "3.9" 
+
+# Installing all the dependencies before testing
+install:
+  - pip3 install -r requirements.txt
+
+# Test command
+script:
+  - py.test -v
```
Comment (anshsrtv) [inline .travis.yml:20]:
```suggestion
  - py.test-3 -v
```

**Review by honeybhardwaj — APPROVED** [review]
LGTM

**Review by anshsrtv — APPROVED** [review]
Nice work writing up the YAML file :tada:

**Comment by aaryanrr** [comment]
All the required changes have been made to the YAML file.

**Pull request merged**